test/sample-data.log

```diff
@@ -0,0 +1,4 @@
+{"ts":1655888896593,"type":"header","entry":{"version":"1.0.0","argv":["C:\\Program Files\\nodejs\\node.exe","C:\\Users\\Dmitriy Petrov\\Desktop\\node-route-metrics-fork\\test\\servers\\express.js"],"node_version":"v16.15.1","os":{"arch":"x64","hostname":"DESKTOP-NK1QURU","freemem":3774451712,"totalmem":8366907392,"getPriority":0,"loadavg":[0,0,0],"uptime":478631,"type":"Windows_NT","platform":"win32","release":"10.0.22000","version":"Windows 10 Pro","userInfo":{"uid":-1,"gid":-1,"username":"Dmitriy Petrov","homedir":"C:\\Users\\Dmitriy Petrov","shell":null},"homedir":"C:\\Users\\Dmitriy Petrov","tmpdir":"C:\\Users\\DMITRI~1\\AppData\\Local\\Temp","endianness":"LE","cpus":8,"cpuModel":"11th Gen Intel(R) Core(TM) i5-11300H @ 3.10GHz","networkInterfaces":{"Wi-Fi":[{"address":"fe80::79e7:65fd:4607:cea3","netmask":"ffff:ffff:ffff:ffff::"},{"address":"192.168.0.104","netmask":"255.255.255.0"}],"Loopback Pseudo-Interface 1":[{"address":"::1","netmask":"ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff"},{"address":"127.0.0.1","netmask":"255.0.0.0"}],"vEthernet (WSL)":[{"address":"fe80::5520:a624:b478:72af","netmask":"ffff:ffff:ffff:ffff::"},{"address":"192.168.112.1","netmask":"255.255.240.0"}]}},"package_json":{"name":"servers","version":"1.0.0","description":"","main":"simple.js","keywords":[],"author":"","license":"ISC"},"config":{"LOG_FILE":"route-metrics.log","OUTPUT_CONFIG":"","GARBAGE_COLLECTION":false,"EVENTLOOP":true,"EVENTLOOP_RESOLUTION":20}}}
```

Comment:
i think the header checks will fail if the rest runs on a different computer. maybe just check for the correct property-names and types and (sometimes) lengths of values?